### PR TITLE
Port Documentation on building WebKit on Windows from Trac

### LINKED ISSUE #44: Port Documentation on building WebKit on Windows from Trac
https://trac.webkit.org/wiki/BuildingCairoOnWindows

### PR DESCRIPTION
**Pull request by fujii**
Imported the Trac Wiki page <https://trac.webkit.org/wiki/BuildingCairoOnWindows>.

Resolves #44

### CODE REVIEW

**Review by donny-dont — APPROVED** [review]
Looks fine though I think you should remove references to AppleWin.

**Comment by fujii** [comment]
Thank you for the review. It's a bit too early. Buildbot is still using the word "WinCairo". I will remove the historical part after the migration completes.

File: docs/Build & Debug/BuildOptions.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Tools/Scripts/build-webkit --wpe --debug
 
 ## Building Windows Port
 
-For building WebKit on Windows, see the [WebKit on Windows page](https://trac.webkit.org/wiki/BuildingCairoOnWindows).
```
Comment (stwrt) [inline docs/Build & Debug/BuildOptions.md:87]: The link to the windows port does not seem to be working from local testing.

http://127.0.0.1:8000/Build%20%26%20Debug/Ports/Windows%20port/Introduction.html

File: docs/Ports/Windows port/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
+# Windows port
```
Comment (stwrt) [inline docs/Ports/Windows port/Introduction.md:1]: Did we need a separate folder for the windows port?

File: docs/Build & Debug/BuildOptions.md (hunk 1 above)
Comment (fujii) [reply docs/Build & Debug/BuildOptions.md:87]: It works on my PC. But, `mkdocs build` is reporting warnings. I think I should use absolute paths. Will fix.
> WARNING  -  Documentation file 'Build & Debug/BuildOptions.md' contains a link to 'Ports/Windows port/Introduction.html' which is not found in the documentation files.
> WARNING  -  Documentation file 'Ports/Windows port/Introduction.md' contains a link to 'Build & Debug/DebuggingWithVS.html' which is not found in the documentation files.

File: docs/Ports/Windows port/Introduction.md (hunk 2 above)
Comment (fujii) [reply docs/Ports/Windows port/Introduction.md:1]: OK. Will fix.

Comment (fujii) [reply docs/Ports/Windows port/Introduction.md:1]: https://github.com/WebKit/Documentation/pull/56

**Pull request merged**